File: Csoport_16_00/3. hét/Movie/Movie.java
public class Movie {
    // Ezeket mind priváttá kell tenni és írni hozzá getter/settert
    public String title;
    public int releaseYear;
    private double score;

    public Movie(String title, int releaseYear, double score) {
        this.title = title;
        this.releaseYear = releaseYear;
        this.setScore(score);
    }

    public void setScore(double value) {
        if (value < 0)
            this.score = 0;
        else if (value > 10)
            this.score = 10;
        else 
            this.score = value;
    }

    public double getScore() {
        return this.score;
    }

    @Override
    public String toString() {
        //  Movie(title (rYear), score)
        return String.format("Movie(%s (%d), %.1f)", this.title, this.releaseYear, this.score);
    }
}

File: Csoport_16_00/12. hét/Osztalyok/Movie.cs
namespace Osztalyok;

public class Movie
{
    private string _title;
    // private int _year;
    private double _score;

    public Movie(string title, int year, double score)
    {
        _title = title;
        //_year = year;
        Year = year;
        _score = score;
    }

    /*public string GetTitle()
    {
        return _title;
    }

    public void SetTitle(string val)
    {
        this._title = val;
    }*/

    public string Title
    {
        get
        {
            return _title;
        }
        private set
        {
            this._title = value;
        }
    }

    public int Year { get; set; }

    public override string ToString()
    {
        return string.Format("Movie({0} ({1}), {2})", Title, Year, _score);
    }
}